pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/SumAggregationFunction.java

```diff
@@ -85,6 +86,15 @@ public void aggregate(int length, AggregationResultHolder aggregationResultHolde
         }
         break;
       }
+      case BIG_DECIMAL: {
```

Comment:
Suggest adding a query execution test for sum/min/max on BigDecimal in any of the segment or inter-segment query execution tests

Reply:
Covered by updated `RealtimeClusterIntegrationTest`.